### PR TITLE
feat(Cards): add IsClickable property

### LINKED ISSUE #465: [Material][Card][CardContentControl] Add IsClickable to enable/disable pointer visual states
Cards should have an `IsClickable` type of property that can drive whether or not we displaying any sort of visual change on pointer events (PointerOver, Pressed, Hover, etc.)

The Material Components Library has an `IsClickable` property to drive this type of behavior:. Example below:

https://user-images.githubusercontent.com/4793020/216088819-55f6ce56-1616-4d36-bf29-f73b63331a74.mp4

### PR DESCRIPTION
**Pull request by Xiaoy312**
GitHub Issue (If applicable): closes #465

## PR Type

What kind of change does this PR introduce?
- Feature

## What is the current behavior?
Card and CardContentControl always respond to pointer & focus events, with no way of disabling this behavior.

## What is the new behavior?
This behavior can now be toggled with the new `IsClickable` property on both controls, that is set to `true` by default.

## PR Checklist
Please check if your PR fulfills the following requirements:
- [x] Tested code with current [supported SDKs](../README.md#supported)
- [x] Docs have been added/updated which fit [documentation template](https://github.com/nventive/Uno/blob/master/doc/.feature-template.md). (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](doc/articles/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] [Wasm UI Tests](doc/articles/working-with-the-samples-apps.md#running-the-webassembly-ui-tests-snapshots) are not showing unexpected any differences. Validate PR `Screenshots Compare Test Run` results.
- [x] Contains **NO** breaking changes
- [x] Updated the [Release Notes](https://github.com/nventive/Uno/tree/master/doc/ReleaseNotes)
- [x] Associated with an issue (GitHub or internal)

## Other information
Internal Issue (If applicable):
closes unoplatform/uno.chefs#113

### CODE REVIEW

**Comment by Xiaoy312** [comment]
@carldebilly @Robert-Louis FYI, a new `IsClickable` property will be added to `Card` and `CardContentControl`, possibly something we want to edit from Figma too.

File: src/Uno.Toolkit.UI/Controls/Card/Card.Properties.cs [hunk 1]
```diff
@@ -14,158 +14,264 @@
...
+		#endregion
+		#region DependencyProperty: IsClickable = true
+
+		public static DependencyProperty IsClickableProperty { get; } = DependencyProperty.Register(
+			nameof(IsClickable),
+			typeof(bool),
+			typeof(Card),
+			new PropertyMetadata(true));
+
+		/// <summary>
```
Comment (agneszitte-nventive) [inline src/Uno.Toolkit.UI/Controls/Card/Card.Properties.cs:266]: If we had a summary here, can we had also a summary for the other properties, please?

**Review by kazo0 — APPROVED** [review]

File: src/Uno.Toolkit.UI/Controls/Card/Card.Properties.cs (hunk 1 above)
Comment (Xiaoy312) [reply src/Uno.Toolkit.UI/Controls/Card/Card.Properties.cs:308]: done

**Pull request merged**